### ISSUE TITLE
[SECURITY ISSUE] Plugin uses commons-collections-3.2.1.jar

### ISSUE THREAD

**Issue by Kurisutian** [rundeck-plugins/jira-workflow-step#2]
The plugin uses an old version of commons-collections which according to the website https://commons.apache.org/proper/commons-collections/security-reports.html contains a security whole which has been fixed in version 3.2.2
It should be replaced with this patched version.

**Comment by marcjay** [comment]
I don't think this is accurate? Or at least not any more. The dependency on commons-collections:3.2.2 is from rundeck-core:2.11.4 - only a runtime dependency. This plugin doesn't bring in commons-collections itself, so I think this should be closed?
